packages/dx-scheduler-core/src/plugins/week-view/helpers.test.ts

```diff
@@ -111,45 +165,102 @@ describe('Week view helpers', () => {
     });
 
     describe('#reduceAppointmentByDayBounds', () => {
+      const cellDuration = 30;
       it('should crop appointment start', () => {
```

Comment:
Maybe it would be better to use 'cut' instead of 'crop' in all the test for this function